core/modules/monkeymsal/private/Get-OsInfo.ps1

```diff
@@ -28,7 +28,7 @@
         #Set ScriptBlock
         $ScriptBlock = {
             try{
-                $files = $args[0]
+                $files = $args
```

Comment:
does this work on both Ubuntu and Windows? Or do we needs to add a an if statement here?

Reply:
This works on both.